### ISSUE TITLE
Tokenization functors should behave and be called like in C

### ISSUE THREAD

**Issue by h-2** [seqan/seqan#879]
We have our own functors for tokenization which are apparently faster, because of inlining. However they are called differently then in ctype.h or are called the same, but do something differently, e.g. our `IsSpace` checks for different characters than `isspace()`. 

This is a table with the functions in the libc: http://www.cplusplus.com/reference/cctype/

This should be fixed before we add documentation for the new IO.

**Comment by h-2** [comment]
I will fix it sometime soon, unless someine objects.

**Comment by esiragusa** [comment]
Well I object. Our IsSpace is meant to check only the *space* character (0x20) while isspace() checks a set of white-space characters (space, tab, etc). We want to have IsSpace as it is and it's name is appropriate as it checks for the *space* character only. Instead, the name of isspace() is misleading and should be called iswhitespace() :trollface:

**Comment by esiragusa** [comment]
By the way, you're welcome to report other inconsistencies...

**Comment by h-2** [comment]
I agree the standardized version is not ideally named, but it would be very helpful, if we stuck to standardized names to avoid confusion. This way we can link to offical documentation and say that our functions behave accordingly.
Otherwise we have to write lots of dox explaining why we created Functors to replace existing functions that we called like the existing functions but then also made them behave differently, because *we* thought it was better. IMHO not a good idea.
